src/khoj/processor/conversation/offline/chat_model.py

```diff
@@ -156,6 +158,16 @@ def converse_offline(
     # Initialize Variables
     compiled_references_message = "\n\n".join({f"{item}" for item in references})
 
+    system_prompt = ""
```

Comment:
nit: this initialization isn't required as system_prompt is always set in the if/else below